editor/src/clj/editor/workspace.clj

```diff
@@ -136,6 +139,8 @@ ordinary paths."
                        :dependencies-fn dependencies-fn
                        :write-fn write-fn
                        :read-fn read-fn
+                       :read-raw-fn (or read-raw-fn read-fn)
+                       :sanitize-fn sanitize-fn
```

Comment:
We now expose the `:sanitize-fn` in the `resource-type` map. We use it in a few places in this PR.